tasks/html/easy/navigation.html

```diff
@@ -33,10 +33,12 @@
     </style>
   </head>
   <body>
-    <!-- TODO:
-      - Add a <nav> element.
-      - Inside the <nav> element, use an unordered list <ul> with list items <li> for each navigation link.
-      - Each list item should contain an anchor <a> element pointing to #, as we don't have actual pages to link to (e.g., <a href="#">Home</a>).
-    -->
+    <nav>
+      <ul>
+      <li><a href="#">Home</a>)</li>
+      <li><a href="#">About</a>)</li>
+      <li><a href="#">Contact</a>)</li>
```

Comment:
This looks mostly good, there are just those extra `)` characters between the `a` and `li` closing tags which you probably put there by accident, but that's fine.